source/directives/uiTreeNode.js

```diff
@@ -572,6 +572,12 @@
                 }
               };
             })();
+            
+            keydownHandler = function (e) {
+              if (e.keyCode == 27) {
```

Comment:
I know the original code also contained `==`, but please use the triple equal sign as much as possible.